src/typings/index.d.ts

```diff
@@ -0,0 +1,34 @@
+type PokemonCard = {
+  id: string
+  name: string
+  supertype: string
+  types: string[]
+  images: {
+    small: string
+    large: string
+  }
+  set: {
+    releaseDate: string
+  }
+  flavorText: string
+  tcgplayer?: {
+    url: string
+  }
+}
+
+type ListCardsResponse = {
+  data: PokemonCardObject[]
+  page: number
+  pageSize: number
+  count: number
+  totalCount: number
+}
+
+type Collection = {
+  cards: PokemonCard[]
+}
+
+type Action = {
```

Comment:
```suggestion
type CollectionAction = {
```

I would be more specific with this name and define it alongside your reducer